pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1320,7 +1320,7 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		domain.Spec.Devices.Controllers = append(domain.Spec.Devices.Controllers, api.Controller{
 			Type:  "scsi",
 			Index: "0",
-			Model: "virtio-scsi",
+			Model: translateModel(c, "virtio"),
```

Comment:
are virtio-scsi  and virtio models the same thing?

Reply:
Yes, for reasons I don't know, the `virtio` model on the scsi controller was called `virtio-scsi`, but it is `virtio` and now the two new names `virtio-transitional` and `virtio-non-transitional` exist for it.

Reply:
The reason is, you guessed it, backwards compatibility ;)

Basically for any other virtio device, for example virtio-blk:

* model=virtio => virtio-blk-pci
* model=virtio-transitional => virtio-blk-pci-transitional
* model=virtio-non-transitional => virtio-blk-pci-non-transitional

But for virtio-scsi:

* model=**virtio-scsi** => virtio-scsi-pci
* model=virtio-transitional => virtio-scsi-pci-transitional
* model=virtio-non-transitional => virtio-scsi-pci-non-transitional

Now that we always use either model=virtio-transitional or model=virtio-non-transitional, we no longer need to special-case virtio-scsi.

Reply:
@davidvossel would you mind approving if that sounds good to you?